packages/user/Host/plugin/crypto/wit/deps/supervisor.wit

```diff
@@ -0,0 +1,27 @@
+package supervisor:bridge;
+
+interface intf {
+    use host:types/types.{pem, error};
+
+    // The top-level app through which the app is accessing psibase
+    //
+    // Cannot be derived from the callstack, as the supervisor is allowed
+    //   to originate calls in the context of a non-supervisor root window
+    //   (e.g. start-tx, finish-tx)
```

Comment:
What?

Reply:
Not completely

Reply:
You talking about the first line, describing Supervisor? If so, i've removed that as well (in the latest commit).